packages/components/src/box-control/utils.ts

```diff
@@ -117,13 +118,17 @@ export function getAllUnitFallback( selectedUnits ) {
 /**
  * Checks to determine if values are mixed.
  *
- * @param {Object} values        Box values.
- * @param {Object} selectedUnits Box units.
- * @param {Array}  sides         Available box sides to evaluate.
+ * @param  values        Box values.
+ * @param  selectedUnits Box units.
+ * @param  sides         Available box sides to evaluate.
  *
- * @return {boolean} Whether values are mixed.
+ * @return Whether values are mixed.
  */
-export function isValuesMixed( values = {}, selectedUnits, sides = ALL_SIDES ) {
+export function isValuesMixed(
+	values: BoxControlValue = {},
+	selectedUnits?: BoxControlValue,
```

Comment:
I marked some of these as optional to match how the functions were actually being used in the code. For example:

https://github.com/WordPress/gutenberg/blob/1aeed0c852ab40f4ab76687a7035bcd9134a6a87/packages/components/src/box-control/index.tsx#L98